src/pages/ProductPage.jsx

```diff
@@ -19,9 +21,25 @@ const ProductPage = () => {
 
   const { productId } = useParams(); // Object Destructuring
 
+  function handleShare() {
+    alert("Share this experience");
+  }
+  function handleSave() {
+    alert("Save this experience");
+  }
+
   return (
     <div>
       {`Dummy Product Page ${productId}`}
+      <div>
+        <IconButton
+          faIcon={faArrowUpFromBracket}
+          label="Share"
+          onClick={handleShare}
+        />
+        <IconButton faIcon={faHeart} label="Save" onClick={handleSave} />
```

Comment:
I would think of the component being,  both the share and the save  (considering the sizes of the other components) 
 Then in the product page we wont have to render the same element twice as it beats the point of reusable components architechture.

Reply:
My idea here is that the icon button is the reusable component. They don't necessarily always go together - for example there is a share component on home page.

Reply:
Just seen, thought we would put them together already as in the screenshot. 
But for the purpose of the icon,  the task is well completed!